scripts/onboarding/aks/azuremonitorprofile/armTemplates/existingClusterParam.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "aksResourceId": {
+      "value": "/subscriptions/<SubscriptionId>/resourcegroups/<ResourceGroup>/providers/Microsoft.ContainerService/managedClusters/<ResourceName>"
+    },
+    "aksResourceLocation": {
+      "value": "<aksClusterLocation>"
+    },
+    "workspaceResourceId": {
+      "value": "/subscriptions/<SubscriptionId>/resourceGroups/<ResourceGroup>/providers/Microsoft.OperationalInsights/workspaces/<workspaceName>"
+    },
+    "workspaceRegion": {
+      "value": "<workspaceRegion>"
+    },
+    "enableContainerLogV2": {
+      "value": true
+    },
+    "disableCustomMetrics": {
+      "value": false
+    },
+    "disablePrometheusMetricsScraping": {
+      "value": false

```

Comment:
since with unified profile, recommendation to use AMP for prom metrics and we should make this default to true.